src/neo/SmartContract/Native/Designate/DesignateContract.cs

```diff
@@ -28,7 +29,8 @@ internal override void Initialize(ApplicationEngine engine)
         {
             foreach (byte role in Enum.GetValues(typeof(Role)))
             {
-                engine.Snapshot.Storages.Add(CreateStorageKey(role), new StorageItem(new NodeList()));
+                engine.Snapshot.Storages.Add(CreateStorageKey(role), new StorageItem(BitConverter.GetBytes(0u))); // Same as BigEndian
```

Comment:
We can avoid the current key, and use `FindRange`, what do you think?

Reply:
How to determine `Range`. Read all history and get the last?

Reply:
You can use `SeekDirection.Backward` and find the value from `uint.MaxValue`.

Reply:
Done.